src/main/java/seedu/address/model/date/BirthDate.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.model.date;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class BirthDate extends DocumentedDate {
+    private static final DateTimeFormatter FORMATTER_INPUT = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+    public final String value;
```

Comment:
Maybe you can consider shifting these two members to the super class?

Reply:
I'll keep these two members in this class because I need to specify the BirthDate class in usage